entity-framework/core/modeling/backing-field.md

```diff
@@ -49,12 +49,8 @@ You can give EF the name of the field in the `Property(...)` API. If there is no
 
 [!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/BackingFieldNoProperty.cs#Sample)]
 
-You can also choose to give the property a name, other than the field name. This name is then used when creating the model, most notably it will be used for the column name that is mapped to in the database.
-
-[!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/BackingFieldConceptualProperty.cs#Sample)]
```

Comment:
Are we not deleting the backing C# file?

Reply:
We should